### ISSUE TITLE
Dropdown arrow won't open dropdown

### ISSUE THREAD

**Issue by HamAndRock** [patw0929/react-intl-tel-input#365]
Clicking on down arrow, won't open the dropdown.

## Expected Behavior
When i click right on the arrow symbol the dropdown should open.

## Current Behavior
The dropdown won't open

## Steps to Reproduce
1. Click right in middle of dropdown arrow

## Environment

* Version:
v8.0.4
* Browser:
Chrome,

**Comment by nehaljadav** [comment]
This issue is accrues because of 
class in dropdown arrow element,
class in country code element (if separateDialCode is true),
if you remove the class from that particular element then it will work perfectly fine.

**To solve Arrow Dropdown Click**:-
In your CSS file add below CSS to solve issue.

**.intl-tel-input.allow-dropdown .arrow {
    display: none;
}
.intl-tel-input.allow-dropdown .selected-flag:after {
    content: "\25BC";
    font-size: 6px;
    margin-left: 5px;
}
.intl-tel-input.allow-dropdown.expanded .selected-flag:after {    
    content: '\25B2';
}**

**To Solve Country Code Click**:-
After intl-tel-input initialize add below code.

**setTimeout(() => {
    document.querySelector('.intl-tel-input .selected-dial-code').classList.remove("selected-dial-code")
}, 50);**


*If you want to add css for Country Code then you can do like this.
.intl-tel-input.allow-dropdown .selected-flag {
    Your: CSS,
}



![before](https://user-images.githubusercontent.com/64223403/106448450-cbb74100-64a8-11eb-8311-922193c6b2b2.png)
![after](https://user-images.githubusercontent.com/64223403/106448466-d1148b80-64a8-11eb-8f78-839f7f7114f6.png)

**Comment by HamAndRock** [comment]
Yeah i don't use separateDialCode={true}, so i dont hate that class there.
Just the blue section causing the problem margin section registers the click ok.

![image](https://user-images.githubusercontent.com/24235083/106478234-484a1f80-64a9-11eb-96a3-0ea1b6649bd4.png)

**Comment by nehaljadav** [comment]
https://meet.google.com/awq-ekac-zun
Join above meeting I will help you out.

**Comment by justgeek** [comment]
Just add a global css ruel to disable click listeners on arrow class

`.arrow {
     pointer-events: none;
}
`
